enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/task/ClientEvaluationSyncTask.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2022 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.app.onboardingserver.task;
+
+import com.wultra.app.onboardingserver.impl.service.ClientEvaluationService;
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+/**
+ * Task to process client evaluation result with the provider.
+ *
+ * @author Lubos Racansky, lubos.racansky@wultra.com
+ */
+@Component
+public class ClientEvaluationSyncTask {
```

Comment:
🤔 I have created a new sync task because there already are some. But now I wonder whether it isn't better and faster to use an async event listener.

Reply:
This sounds like an improvement, but I would postpone it to later times and discuss all performance optimizations then.